src/chat/ChatScreen.tsx

```diff
@@ -69,7 +67,7 @@ interface ChatScreenProps extends GiftedChatProps {
   typingTimeoutSeconds?: number;
   messageStatusEnable?: boolean;
   customMessageStatus?: (hasUnread: boolean) => JSX.Element;
-  iconsCamera?: IconPaths;
+  children?: (props: ChildrenProps) => ReactNode | ReactNode;
```

Comment:
seem we miss action onPressCamera. Can you check it?

Reply:
removed